packages/cascara/package.json

```diff
@@ -65,6 +65,7 @@
   },
   "peerDependencies": {
     "@espressive/design-tokens": "*",
+    "@espressive/prop-types": "*",
```

Comment:
@lordpixel thank you for doing this.

I am noticing that the scoped dependency above that is existing, and this one, are not following the `workspace:` syntax. We should follow that syntax so we can be accurate with the versions expected, instead of using the wildcard, which can bite us in the butt later if people do not update their versions.

Could we make a change to this to change any of our Cascara monorepo peerDeps to make sure they are not using the wildcard style syntax and instead are using the yarn workspace syntax?

Reply:
That's how we did it:
![image](https://user-images.githubusercontent.com/13924868/148292654-681e79fd-774d-407c-ae94-beeb2707bcf6.png)

I don't know why it was changed..

Reply:
trying again..

Reply:
That is how we did it, but I think `yarn` is chaging it...

#### Before install
![image](https://user-images.githubusercontent.com/13924868/148293540-3879a416-997e-4b4e-8f96-3c0a7c8f79ac.png)

#### After install
![image](https://user-images.githubusercontent.com/13924868/148293773-f802e0e2-2ddd-4381-869a-0390214bc056.png)

Reply:
Hmm. I will take a look at this too.